cli/src/main.rs

```diff
@@ -145,6 +97,11 @@ fn install_tracing_subscriber(opts: &Opts) {
             }
         }
     }
+    // If the quiet flag is passed in, don't log anything (this should maybe
+    // just be a verbosity setting)
```

Comment:
I think your instinct is right here. `-v0` should be quiet, `-v1` should be what is currently the default with no `-v` arguments, and so on. This probably means changing around the definition of the `-v` flag.

The default for `run` mode should be `-v0`.

Reply:
Let's save this for the follow-up PR reworking our CLI interface